larray_editor/arrayadapter.py

```diff
@@ -36,7 +36,7 @@ def get_axes_names(self):
 
     def get_axes(self):
         axes = self.filtered_data.axes
-        if len(axes) == 0:
+        if self.filtered_data.size == 0 or len(axes) == 0:
```

Comment:
`or len(axes) == 0` seems superfluous. I can see how you can have `size == 0` with `len(axes) > 0`, but I don't see any case where you can have `size > 0` with `len(axes) == 0`. You could also write: `if not self.filtered_data.size:` but that is a matter of taste.